core/modules/layout/includes/block.class.inc

```diff
@@ -426,6 +435,53 @@ class Block extends LayoutHandler {
   function getChildren() {
     return NULL;
   }
+
+  /**
+   * Get a render array for a block.
+   */
+  function getRenderArray() {
```

Comment:
Let's call this `view()` so it matches the general approach with entities, like with `Node::view()`. I don't think including the variable type in the name follows Backdrop convention.

Reply:
I realize that there's also `hook_block_view` which is used by the BlockLegacy class to build the content part. And here this takes that content render array and outputs another content array.

Reply:
Or maybe `buildBlock()`.